src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -419,3 +422,13 @@ def _rand_str(n):
     """
     choices = string.ascii_lowercase + string.digits
     return ''.join(random.SystemRandom().choice(choices) for _ in range(n))
+
+def _mkdir_p(path):
+    # http://stackoverflow.com/a/600612
+    try:
+        os.makedirs(path)
+    except OSError as exc:  # Python >2.5
```

Comment:
I think the comment was saying that OSError is only thrown from `os.makedirs()` on Python 2.5 and up.